### PR TITLE
Add pagination for books and reviews

### PR DESCRIPTION
**Pull request by giftnichakul**
- pagination for books ( 5 books per page)
  image
- pagination for reviews ( 10 reviews per page)
  image
  image

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -3,10 +3,12 @@ class BooksController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @books = Book.all
+    @books = Book.page(params[:page]).per(5)
```
Comment (kanatsanan6) [inline app/controllers/books_controller.rb]: if we actually want to hard-coded a `per` params (which normally we don't). we can specify `paginates_per ` on each model.

ref: https://github.com/kaminari/kaminari#configuring-default-per_page-value-for-each-model-by-paginates_per

**Review by kanatsanan6 — APPROVED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -8,4 +8,7 @@
   resources :books do
     resources :reviews
   end
+
+  # resources :books, only: [:index]
+  # resources :reviews, only: [:index]
```
Comment (WatcharitNew) [inline config/routes.rb]: do not forget to remove comments

File: app/controllers/books_controller.rb [hunk 3]
```diff
@@ -3,10 +3,12 @@ class BooksController < ApplicationController
...
   end
 
-  def show; end
+  def show
+    @reviews = @book.reviews.page(params[:page]).per(10)
```
Comment (WatcharitNew) [inline app/controllers/books_controller.rb]: also this one you can follow https://github.com/giftnichakul/practice_ruby_on_rails/pull/11/files#r1223856866

**Review by WatcharitNew — APPROVED** [review]

**Comment by WatcharitNew** [comment]
if possible, please also add test to check that the records size is equal to per page that we setup or not

**Pull request merged**